common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -48,19 +41,233 @@ Inclusion and Exclusion Fences are defined easily in Mission Planner under the P
+distance, despite the backup fences, the vehicle will switch into LAND mode (HOLD for Rover).  The idea being that it's clearly impossible to get the vehicle home so best to just bring it
+down/stop it.  The pilot can still retake control of course with the flight mode
+switches.  Like with the earlier fences, another fence is erected 20m
+out which will again switch the vehicle to LAND (HOLD for Rover), if it continues away from
+home.
+[/site]
+[site wiki="plane"]
+
+If enabled, and  the fence boundaries or altitudes(:ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` and :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>`), if non-zero,  are exceeded, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below:
 
++------------+--------------------+
++FENCE_ACTION|  Plane             +
++============+====================+
++     0      |   Report Only      +
++------------+--------------------+
++     1      | RTL                +
++------------+--------------------+
++     6      | Guided to return   +
++            | point              +
++------------+--------------------+
++     7      |  Guided to return  +
++            |  point with pilot  +
++            |  throttle control  +
++------------+--------------------+
+
+The return (and loiter) point, unless :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled, is the geometric center of the breached fence boundary. If :ref:`FENCE_ACTION<FENCE_ACTION>` is set to a guided return mode (6 or 7), the altitude at which it returns is determined by :ref:`FENCE_RET_ALT<FENCE_RET_ALT>` or if  :ref:`FENCE_RET_ALT<FENCE_RET_ALT>` is zero, midway between the altitude limits, if non-zero. If both altitude limits are zero, then it will return at current altitude.
+
+The return point can also be replaced by the nearest rally point (See: :ref:`common-rally-points`) as a destination, if :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled.
+
+.. note:: while executing any action that requires a mode change (ie. all except Report Only), any additional fence breaches are recorded but ignored. For example, RTL that has an AutoLand will breach a MIN_ALT fence if enabled, but take no action and finish the landing.
+
+Modes changes cannot be executed after the fence breach unless the :ref:`FENCE_OPTIONS<FENCE_OPTIONS>` bit 0 is set to "0". By default it is set to "1" to prevent mode changes. When set, the vehicle must return to within the fence boundary in order to change modes. If mode changes are allowed, you can also return to the mode that was active upon breach by using an RC auxiliary switch set to option "96" (Re-read mode switch).
+[/site]
+
+Enabling the Fence with an RC Channel Auxiliary Switch
+======================================================
+
+It is not necessary to set-up a switch to enable or disable the fence
+but if you wish to control the fence with a switch follow these
+steps:
+
+An RCx_OPTION can be set via the Config/Tuning > Full Parameter List screen:
+
+-  Use an RCx_OPTION set to Fence
```

Comment:
It might be good to specify the option number.  I'd personally add a link back to the aux function wiki page.

Reply:
ok